backend/_example/memory_store/server/rpc_test.go

```diff
@@ -401,7 +401,13 @@ func prepTestStore(t *testing.T) (s *RPC, port int, teardown func()) {
 		log.Printf("%v", s.Run(port))
 	}()
 
-	time.Sleep(time.Millisecond * 50) // wait for server to start
+	// wait for server to start before returning it
+	for {
```

Comment:
the endless loop looks to ... radical. Need an exit (with test fail) if not started for whatever reason in 1-2 seconds or so

Reply:
Good point, adjusted it to be limited by 3 seconds.